### PR TITLE
feat: adding debug logging commands

### PR DESCRIPTION
**Pull request by datascienceChris**
### Link to Github Issue
---
https://github.com/hipagesgroup/data-tools/issues/136

### What changes are proposed in this PR?
---
Adding logging statements to the athena -> adwords modules to help with debugging.

### How was this tested?
---
Unit tests

### CODE REVIEW

File: hip_data_tools/etl/athena_to_adwords.py [hunk 1]
```diff
@@ -123,8 +123,14 @@ def _upload_conversions(self, data_batch):
         return self._get_adwords_util().upload_conversions(data_batch)
 
     def _state_manager_connect(self):
+
+        LOG.debug("Connecting to Cassandra")
+
         conn = CassandraConnectionManager(self.__settings.etl_state_manager_connection)
         conn.setup_connection(self.__settings.etl_state_manager_keyspace)
+
+        LOG.debug(f"Cassandra connection established: {conn.get_cluster()}")
```
Comment (Gurado) [inline hip_data_tools/etl/athena_to_adwords.py:132]: Should this maybe be `Log.info`. It's usually only done once at the beginning. @datascienceChris

File: hip_data_tools/etl/athena_to_adwords.py [hunk 2]
```diff
@@ -146,20 +152,32 @@ def _mark_upload_results(self, fail: List[dict], success: List[dict]) -> None:
...
     def _sanitise_data(self, dat):
         try:
             if self._verify_state(dat):
+
+                LOG.debug(f"State verified with, data: \n {dat} \n")
                 return dat, None
             else:
+
+                LOG.debug(f"Sink state found to be not ready, the data is: \n"
+                          f" {dat} \n")
```
Comment (vbmudalige) [inline hip_data_tools/etl/athena_to_adwords.py:171]: why do we need a \n at the end?
`  f" {dat} \n")`

File: hip_data_tools/etl/common.py [hunk 3]
```diff
@@ -75,16 +77,27 @@ def __init__(self, record_identifier: str, etl_signature: str):
         self.remote_state = self._get_or_create_state()
 
     def _get_or_create_state(self):
+
         try:
+
+            LOG.debug(f"Creating ETL record sink using etl_signiture: "
```
Comment (Gurado) [inline hip_data_tools/etl/common.py:83]:
```suggestion
            LOG.debug(f"Creating ETL record sink using etl_signature: "
```

**Review by vbmudalige — APPROVED** [review]
LGTM

**Pull request merged**